plugins/node/opentelemetry-instrumentation-fastify/src/instrumentation.ts

```diff
@@ -102,6 +102,22 @@ export class FastifyInstrumentation extends InstrumentationBase {
     };
   }
 
+  private _hookOnError() {
+    const instrumentation = this;
+
+    return (request: any, reply: any, error: Error, done: () => void) => {
+      if (!instrumentation.isEnabled()) {
+        return done();
+      }
+
+      const onError = instrumentation.getConfig().onError;
+      if (onError) {
+        onError(request, reply, error);
+      }
+      done();
+    };
+  }
```

Comment:
Does not re-throwing the error here change fastify behvaior? ("swallowing"  error for other handlers?)

Reply:
good point, I will double check how this is supposed to behave/work!

Reply:
So i verified that just calling `done()` will bubble the error up to the next hook:

```js
fastify.addHook('onError', (request, reply, error, done) => {
  console.log('hook 1');
  done();
});

fastify.addHook('onError', (request, reply, error, done) => {
  console.log('hook 2');
  done();
});

app.setErrorHandler(function (error, request, reply) {
 console.log('error handler');
  reply.status(409).send({ ok: false })
});
```

When an error happens, results in:

* `hook 1`
* `hook 2`
* `error handler`

I also verified that as soon as a user adds an error handler via `setErrorHandler()`, errors are not bubbled up anymore to global error handlers.